docs/cip/README.md

```diff
@@ -37,231 +58,888 @@ Leios achieves its scalability through a decoupled block production and aggregat
-| Expiration of unreferenced endorser blocks | $r_\text{EB}$    | slot     |                                                                             |                            |                                                              |
-| Number of shards                           | $k_\text{shard}$ | unitless |                                                                             |                            |                                                              |
-| Mean committee size                        | $n$              | parties  |                                                                             |                            |                                                              |
-| Quorum size                                | $\tau$           | parties  |                                                                             | $\tau > n / 2$             |                                                              |
-| . . .                                      |                  |          |                                                                             |                            |                                                              |
-| Network diffusion time                     | $\Delta$         | slot     | Upper limit on the time needed to diffuse a message to all nodes.           | $\Delta > 0$               | Messages have a finite delay.                                |
-| Praos active slot coefficient              | $f_\text{RB}$    | 1/slot   | The probability that a party will be the slot leader for a particular slot. | $0 \lt f \leq \Delta^{-1}$ | Blocks should not be produced faster than the network delay. |
+Nodes receiving the RB header discover the announced EB and fetch its content. The EB contains references to transactions. If a node does not already possess a transaction referenced in the EB, it explicitly requests that transaction from peers.
 
-### Specification for votes and certificates
+**Step 3: Committee Validation**
+
+A **voting committee** of stake pools validates the EB within a time window
+called the **stage length** ($L$):
+
+- Committee members are selected via sortition (lottery based on stake)
+- A committee member votes positively for an EB only if:
+  - It has received the EB within $L_\text{vote}$ slots from its creation,
+  - The EB corresponds to the EB announced in the latest block in the chain maintained by the party,
+  - The transactions in the EB are a valid extension of the RB (and the corresponding chain) that announced it.
+- The stage has two phases: voting period ($L_\text{vote}$) + vote diffusion period ($L_\text{diff}$)
+
+**Step 4: Certification**
+
+If enough committee votes are collected such that the total stake exceeds a
```

Comment:
Is there ambiguity in the term "total stake"? Would "total *active* stake" be more precise? Stake for retired pools should not be in the total, for instance.